lib/ts/recipe/emailpassword/types.ts

```diff
@@ -254,6 +292,7 @@ export type TypeEmailPasswordPasswordResetEmailDeliveryInput = {
     type: "PASSWORD_RESET";
```

Comment:
the email delivery implementations should use the recipeUserId instead of user.id in its implementation

Reply:
what if we don’t change the user object on recipe level at all. So it has no recipeId and the id in that is always recipeId just like before